app/js/utils/location.js

```diff
@@ -0,0 +1,21 @@
+import {API_KEY} from "../features/metadata/constants";
+
+const geocoder = require('google-geocoder');
+
+export function getLocation(place) {
+  return new Promise((resolve, reject) => {
+    const geo = geocoder({
+      key: API_KEY
+    });
```

Comment:
Not sure if really valuable but that doesn't need to be instantiate all the time and that could be out of the function?